install.js

```diff
@@ -2,16 +2,16 @@
 
 var fs = require("fs");
 var os = require("os");
-const {encode: encodeQuery} = require('querystring')
-const {strictEqual} = require('assert')
+const { encode: encodeQuery } = require('querystring')
```

Comment:
Please don't change the code formatting here. It makes both reviewing this PR and Git-blaming harder.

Reply:
Not my intention to inherently format it otherwise so I've reverted these formatting issues now, thanks for pointing it out